.rubocop.yml

```diff
@@ -0,0 +1,54 @@
+Documentation:
+  Enabled: true
+
+AllCops:
+  Include:
+    - "**/Rakefile"
+    - "**/config.ru"
+  Exclude:
+    - "db/**/*"
+    - "tmp/**/*"
+    - "vendor/**/*"
+    - "bin/**/*"
+    - "log/**/*"
+  RunRailsCops: true
+
+Documentation:
+  Enabled: false
+Style/LineLength:
+  Max: 145
+Metrics/AbcSize:
+  Max: 30
+Metrics/BlockNesting:
+  Max: 3
+Metrics/ClassLength:
+  CountComments: false  # count full line comments?
+  Max: 100
+Metrics/CyclomaticComplexity:
+  Max: 6
+Metrics/LineLength:
+  Max: 80
+  AllowURI: true
+  URISchemes:
+    - http
+    - https
+
+Style/ClassAndModuleChildren:
+  Enabled: false
+
+Metrics/MethodLength:
+  CountComments: false  # count full line comments?
+  Max: 13
+
+Metrics/ParameterLists:
+  Max: 5
+  CountKeywordArgs: true
+
+Metrics/PerceivedComplexity:
+  Max: 7
+
+StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
This is the opposite of the default StringLiterals EnforcedStyle value. What is the reasoning behind this?